pkg/util/namespace/namespace_test.go

```diff
@@ -7,73 +7,54 @@ import (
 	"testing"
 )
 
-func TestIsOpenShift(t *testing.T) {
+func TestIsOpenShiftNamespace(t *testing.T) {
 	for _, tt := range []struct {
 		namespace string
 		want      bool
 	}{
 		{
-			want: true,
+			namespace: "openshift-apiserver",
+			want:      true,
 		},
 		{
-			namespace: "openshift-ns",
+			namespace: "openshift-azure-operator",
 			want:      true,
 		},
 		{
-			namespace: "openshift",
+			namespace: "openshift-azure-logging",
 			want:      true,
 		},
 		{
-			namespace: "kube-ns",
-			want:      true,
+			namespace: "openshift-gitops",
+			want:      false,
+		},
+		{
+			namespace: "openshift-authentication",
+			want:      false,
 		},
 		{
 			namespace: "default",
-			want:      true,
+			want:      false,
```

Comment:
I left another comment below, but to reiterate:

Namespace `default` houses the cluster-critical `service/kubernetes` and OpenShift-critical `service/openshift` - has this risk been assessed? It might be good to at least keep Events and audit logs for these Services.

Reply:
Just for history. I think this concern was addressed [here](https://github.com/Azure/ARO-RP/pull/1994#discussion_r865019512).